aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/App.java

```diff
@@ -0,0 +1,48 @@
+package software.amazon.ssm.patchbaseline;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+
+/**
+ * Handler for requests to Lambda function.
+ */
+public class App implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
```

Comment:
why this file?

Reply:
We are using this file to check the health status of API when we initiate the lambda server

Reply:
is this the recommendation, i don't see it's being implemented by other services.

Reply:
I don't see it's being used anywhere in the package, is it used by CFN?

Reply:
I have used this App class for contract test execution and validation . While we run the contract test it will call the this handler and it will check whether the handler are invoking or not.
And we are not using this functionality on other handler.

If we dont need i can remove this file

Reply:
Okay i am removing the App.java file